client/me/purchases/manage-purchase/index.tsx

```diff
@@ -639,6 +642,76 @@ class ManagePurchase extends Component<
 		return null;
 	}
 
+	renderRefundText() {
+		const { purchase, translate } = this.props;
+
+		// Hide if refund window has lapsed.
+		if ( ! purchase?.isRefundable || ! purchase?.refundText || ! purchase?.mostRecentRenewDate ) {
+			return;
+		}
+
+		const ONE_DAY_IN_MILLISECONDS = 24 * 60 * 60 * 1000;
+		const lastRenewalDate = new Date( purchase?.mostRecentRenewDate );
+		const refundPeriodEndDate = new Date(
+			lastRenewalDate.getTime() + purchase?.refundPeriodInDays * ONE_DAY_IN_MILLISECONDS
```

Comment:
Similar to above, I don't think it's possible for `purchase` to be falsy here so we can remove the optional chaining.

```suggestion
			lastRenewalDate.getTime() + purchase.refundPeriodInDays * ONE_DAY_IN_MILLISECONDS
```